src/devices/LidarLiteV3/AcquisitionSettings.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.DistanceSensor.Models.LidarLiteV3
+{
+    /// <summary>
+    /// Various acquisition behavior settings
+    /// </summary>
+    [Flags]
+    public enum AcquisitionSettings
+    {
+        /// <summary>
+        /// Enables reference process during measurement
+        /// </summary>
+        EnableReferenceProcess = 0x40,
+
+        /// <summary>
+        /// Use default delay (10 hz) instead of delay configured in MEASURE_DELAY (0x45) for repetition mode
```

Comment:
```suggestion
        /// Use custom delay instead of default delay
```

I'm not sure what 10Hz delay means? 0.1s?

Reply:
Not certain, but as I understand it, it's how often the sensor fires in a second.  The default delays slows it down enough to have the sensor fire 10 times a second.  A custom delay of 20 would result in 100 sensor readings a second.  I'm fine with the simplified description.